content/en/docs/reference/command-line-tools-reference/feature-gates/anonymous-auth-configurable-endpoints.md

```diff
@@ -9,6 +9,10 @@ stages:
   - stage: alpha 
     defaultValue: false
     fromVersion: "1.31"
+    toVersion: "1.31"
+  - stage: beta 
+    defaultValue: true
+    fromVersion: "1.32"
 ---
 Enable [configurable endpoints for anonymous auth](/docs/reference/access-authn-authz/authentication/#anonymous-authenticator-onfiguration) 
```

Comment:
For beta, we ask for a higher quality bar. Please also fix the link - https://kubernetes.io/docs/reference/access-authn-authz/authentication/#anonymous-authenticator-onfiguration doesn't work - and check that the page this links to would be up to date.

Reply:
> For beta, we ask for a higher quality bar.

Can you elaborate on what you are asking? Is this comment meant to ask me to update the incorrect link?

Reply:
What we (SIG Docs) would like, as part of the graduation to beta, is that you / SIG Auth check through the existing documentation and make sure it's beta quality or better.

This is typically:
- fix any broken links etc
- align with the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) (broadly - doesn't mean get every detail perfect)
- ensure all the associated docs are factually correct

Generally, the docs should be written as if the feature is GA, but then add caveats about enabling behavior if needed.

These could be more than one PR. For this PR, yes: please fix the broken link.

Reply:
Thanks, fixed the broken links and took a pass at https://kubernetes.io/docs/reference/access-authn-authz/authentication/#anonymous-authenticator-configuration, it LGTM, but happy to update it if it doesn't meet the style guide. That is the only doc for this feature, so we should be good to go?